11-3.3-3.3/extra/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM pgrouting/pgrouting:11-3.3-3.3.5
```

Comment:
Is the pgrouting docker image tagged as `pgrouting/pgrouting:11-3.3-3.3.5` or `pgrouting/pgrouting:11-3.3-3.3`?

If it is tagged as the latter one, then we should update the Osm2pgRouting Dockerfile such that it references the correct pgRouting docker image version (that does not contain .patch name). Need to update `update.sh` file to take this into account.

Reply:
We are storing in the repository only pgrouting `major.minor`, but we need to compile the exact version to avoid conflicts when you upgrade the docker image in your server (or local environment). Anyway we continue discussing that in another PR or issue.

Reply:
@krashish8  Thanks for comments and approval!

> Is the pgrouting docker image tagged as `pgrouting/pgrouting:11-3.3-3.3.5` or `pgrouting/pgrouting:11-3.3-3.3`?
> 
> If it is tagged as the latter one, then we should update the Osm2pgRouting Dockerfile such that it references the correct pgRouting docker image version (that does not contain .patch name). Need to update `update.sh` file to take this into account.

It is tagged as the former one (`pgrouting/pgrouting:11-3.3-3.3.5`) from `version.txt` contents.

In the following `Makefile` changes, docker image tag will be created as follows.
1. Create tag from `version.txt` contents (ex. `15-3.3-3.4.1`).
2. If `$1` (=folder name (ex. `15-3.3-3.4`)) doesn't equal `version.txt` contents (ex. `15-3.3-3.4.1`), create another tag (alias) as `15-3.3-3.4`.
```diff
- 	docker build --pull -t $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1) $1
- 	docker build -t $(REPO_NAME)/$(IMAGE_NAME)-extra:$(shell echo $1) $1/extra
+ 	docker build --pull -t $(REPO_NAME)/$(IMAGE_NAME):$(shell cat $1/version.txt) $1
+ 	docker build -t $(REPO_NAME)/$(IMAGE_NAME)-extra:$(shell cat $1/version.txt) $1/extra
+ 	if [ "$(shell echo $1)" != "$(shell cat $1/version.txt)" ]; then\
+ 		docker tag $(REPO_NAME)/$(IMAGE_NAME):$(shell cat $1/version.txt) $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1);\
+ 		docker tag $(REPO_NAME)/$(IMAGE_NAME)-extra:$(shell cat $1/version.txt) $(REPO_NAME)/$(IMAGE_NAME)-extra:$(shell echo $1);\
+ 	fi
```

So, after executing `make build`, created docker images will be as follows.
```sh
$ docker image ls
REPOSITORY                  TAG              IMAGE ID       CREATED        SIZE
pgrouting/pgrouting-extra   15-3.3-main      edc542164c5b   7 hours ago    654MB
pgrouting/pgrouting         15-3.3-main      6798de41297c   7 hours ago    623MB
pgrouting/pgrouting-extra   15-3.3-develop   c44cf12cfe7a   7 hours ago    654MB
pgrouting/pgrouting         15-3.3-develop   258c78de7f1c   7 hours ago    623MB
pgrouting/pgrouting-extra   15-3.3-3.4       e58bdc8a471b   7 hours ago    654MB
pgrouting/pgrouting-extra   15-3.3-3.4.1     e58bdc8a471b   7 hours ago    654MB
pgrouting/pgrouting         15-3.3-3.4       af0216624721   7 hours ago    623MB
pgrouting/pgrouting         15-3.3-3.4.1     af0216624721   7 hours ago    623MB
:
```